x/stakeibc/keeper/icqcallbacks_delegator_shares_test.go

```diff
@@ -348,19 +348,6 @@ func (s *KeeperTestSuite) TestDelegatorSharesCallback_WeightOverfow() {
 	s.Require().Regexp(expectedErrMsg, err.Error())
 }
 
-func (s *KeeperTestSuite) TestDelegatorSharesCallback_SlashGtTenPercent() {
```

Comment:
Should we leave this and just expect success or is that redundant? Maybe we could add a GT 10% example to the success case. Will leave to you.

Reply:
I think we should gut it. Doesn't make sense imo to have this anymore since we don't have specific branching based on the slash magnitude!